packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -604,12 +604,18 @@ class SkiaGoldClient {
 
   /// Returns a boolean value to prevent the client from re-authorizing itself
   /// for multiple tests.
-  bool _clientIsAuthorized() {
+  Future<bool> clientIsAuthorized() async {
     final File authFile = workDirectory?.childFile(fs.path.join(
       'temp',
       'auth_opt.json',
     ));
-    return authFile.existsSync();
+
+    if(await authFile.exists()) {
+      final String contents = await authFile.readAsString();
+      final Map<String, dynamic> decoded = json.decode(contents) as Map<String, dynamic>;
+      return !(decoded['GSUtil'] as bool);
```

Comment:
I'm inclined to leave this here as a back up, we can make sure we aren't caching anything, but if for some reason Gold tries to fallback on gsutil again it should catch it.

Reply:
SG